modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/StandaloneCliOptions.cs

```diff
@@ -161,39 +165,57 @@ public class StandaloneCliOptions : Dictionary<string, string>, IAgentConfigProv
+                    { "ic|iotcentral=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "ns|noshutdown=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "rf|runforever", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "pn|portnum=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "pa|path=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "lr|ldsreginterval=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "ss|suppressedopcstatuscodes=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "csr", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "ab|applicationcertbase64=", "Legacy - do not use.",b => isLegacyOption = true },
+                    { "af|applicationcertfile=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "pk|privatekeyfile=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "pb|privatekeybase64=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "cp|certpassword=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "tb|addtrustedcertbase64=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "tf|addtrustedcertfile=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "ib|addissuercertbase64=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "if|addissuercertfile=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "rb|updatecrlbase64=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "uc|updatecrlfile=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "rc|removecert=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "dt|devicecertstoretype=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "dp|devicecertstorepath=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "i|install", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "st|opcstacktracemask=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "sd|shopfloordomain=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "vc|verboseconsole=", "Legacy - do not use.", b => isLegacyOption = true },
+                    { "as|autotrustservercerts=", "Legacy - do not use.",b => isLegacyOption = true }
                 };
 
-            options.Parse(args);
+            try {
+                options.Parse(args);
+            }
+            catch (Exception e) {
+                logger.Warning("Parse args exception: " + e.Message);
+                Environment.Exit(0);
```

Comment:
We should have non-zero exit code in this case to indicate error. Maybe we can use `160` to follow guidelines [here](https://docs.microsoft.com/en-us/windows/win32/debug/system-error-codes--0-499-?redirectedfrom=MSDN).